mlflow/tracking/fluent.py

```diff
@@ -1072,6 +1072,59 @@ def log_table(
     MlflowClient().log_table(run_id, data, artifact_file)
 
 
+@experimental
+def load_table(
+    artifact_file: str,
+    run_ids: Optional[List[str]] = None,
+    extra_columns: Optional[List[str]] = None,
+) -> "pandas.DataFrame":
+    """
+    Load a table from MLflow Tracking as a pandas.DataFrame. The table is loaded from the
+    specified artifact_file in the specified run_ids. The extra_columns are columns that
+    are not in the table but are augmented with run information and added to the DataFrame.
+
+    :param artifact_file: The run-relative artifact file path in posixpath format to which
+                          the table is saved (e.g. "dir/file.json").
+    :param run_ids: Optional list of run_ids to load the table from. If no run_ids are specified,
+                    the table is loaded from all runs in the current experiment.
```

Comment:
This can be expensive. We might want to add a limit for the number of artifacts to load in the future.

Reply:
Yes agreed.